main.go

```diff
@@ -61,32 +60,21 @@ func init() {
 }
 
 func main() {
-	bindAddr := os.Getenv("BIND_ADDR")
-	if len(bindAddr) == 0 {
-		bindAddr = ":20010"
-	}
 
-	var err error
-	config.DebugMode, err = strconv.ParseBool(os.Getenv("DEBUG"))
+	log.Namespace = "dp-frontend-renderer"
```

Comment:
Slightly out of scope but while you are here is it worth implementing the `run()` pattern? Can have a single exit point then.

Reply:
Yer this is a out of scope but if not done guess it never will be. I will ask if we can get a ticket drawn up for this task of configs as this has indeed escalated way out of scope of the original task.

Reply:
@dandwal Happy for us to adopt this pattern, however if you do then I would recommend doing so in a new package (maybe `service`), so that you have something like `service.run()`. This pattern will be more testable and it is usually best practice to avoid adding functions to the `main` package.